### ISSUE TITLE
okc dataset is no longer in modeldata package

### ISSUE THREAD

**Issue by wnarifin** [skeydan/Deep-Learning-and-Scientific-Computing-with-R-torch#14]
Hi, 

Firstly, thanks for a very nice book and sharing it for free.

In subsection **3.2.3 Tensors from datasets**, you used **okc** dataset from **modeldata**. Of note, the dataset is no longer in the latest version of **modeldata** (version 1.2.0). Maybe you want to consider changing the dataset.

Best regards

**Comment by RodneyPhJ** [comment]
Yes, I encountered the same problem. Is the dataset available elsewhere?
Otherwise I am enjoying the book.
best regards.

**Comment by tenomoto** [comment]
It was deplicated in 0.1.1 and removed entirely from 0.1.2.
https://github.com/tidymodels/modeldata/pull/29

**Comment by ULK-KU** [comment]
Install the previous version via
`remotes::install_github("tidymodels/modeldata@v0.1.1")`
